tests/pl/test_render_shapes.py

```diff
@@ -456,6 +456,86 @@ def test_plot_can_do_non_matching_table(self, sdata_blobs: SpatialData):
 
         sdata_blobs.pl.render_shapes("blobs_circles", color="instance_id").pl.show()
 
+    def test_plot_can_color_with_norm_no_clipping(self, sdata_blobs: SpatialData):
+        blob = deepcopy(sdata_blobs)
+        blob["table"].obs["region"] = "blobs_polygons"
```

Comment:
You create the same data obj in all tests, please move to conftest and get it from there